util/chunk/column.go

```diff
@@ -628,3 +628,14 @@ func (c *Column) CopyReconstruct(sel []int, dst *Column) *Column {
 	}
 	return dst
 }
+
+// OrNulls does the OR operation with all columns in the arguments.
+// The user should ensure that all these columns have the same length, and
+// data stored in these columns are fixed-length type.
+func (c *Column) OrNulls(cols ...*Column) {
```

Comment:
Add a unit test?

Reply:
My mistake and a unit test is added now.